example/android/local.properties

```diff
@@ -0,0 +1,5 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Should you check this in?

Reply:
oh i see what you mean, i'll change that wording